### DISCUSSION TITLE
Cisco Wifi MAC Address filter - iPhone bypass?

### DISCUSSION THREAD

**Discussion by md4r2** [3kh0/ext-remover#1035] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So a few years ago I used luphoria's policy exploit to find my organisation's wifi password. Soon the whole place found out, which was great, but (only) now the IT department have found out. They added some kind of mac address filter by cisco which is probably a whitelist. 

There's no issue for chromebooks because I can just sudo ifconfig and temporarily change the mac. Also, for most android phones, there is a dev option with persistent MAC randomisation so they're fine too.

 The only problem is with iPhones because as far as I can tell, the only option is to switch off random MAC (so phone MAC is enabled), however somehow the IT is easily blocking these as well. After that, switching back to randomised MAC doesn't actually RErandomise it, so its the same as before, so it stays blacklisted. Just for clarification, iPhone users can still connect to the network, but internet access is denied, so VPNs obviously don't work.

Anyone got any bypasses, tips or advice? Would appreciate any at all ASAP, half the place has gone nuts.

**Comment by Inglan2** [comment, marked as answer]
Don't think this is possible
[discussions.apple.com](https://discussions.apple.com/thread/8202864?sortBy=best#:~:text=2%3A02%20PM-,If%20the%20Access%20Control%20in%20the%20router%20has%20been%20set%20to%20allow%20only%20certain%20devices%20and%20the%20phone%20in%20question%20isn%27t%20one%20of%20them%20(or%20to%20block%20the%20phone%27s%20MAC%20address)%2C%20no%20amount%20of%20rebooting%20the%20phone%20or%20disconnecting%20and%20reconnecting%20from%20the%20wifi%20will%20make%20any%20difference.,-View%20in%20context)

**Reply by md4r2** [reply]
Thanks for the reply
